### PR TITLE
Contributions Doc

### PR DESCRIPTION
**Pull request by katyhuff**
Adds some richness to the contributions doc. Still not enough to constitute a fix to open issues related to node guidelines, but helpful, perhaps, nonetheless. Covers basics of git and borrows some links, hints, and workflow lists from cyclus/cyclus CONTRIBUTING.rst as well as other contributing.md docs around the net.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+
+
+Start by copying the template node directory and work from there. Beyond that,
+the process will follow the [contributor workflow.](#contributor-workflow)
+
+## Contributor Workflow
+
+- The terminology we use is based on the [Integrator Workflow](http://en.wikipedia.org/wiki/Integrator_workflow).
+- The first step is to create your own [fork](https://help.github.com/articles/fork-a-repo/) of the project.
+- Use a branching workflow similar to the one described in the [pro-git book](http://progit.org/book/ch3-4.html).
```
Comment (TheDoctorRAB) [inline CONTRIBUTING.md]: This link doesn't work for me.

**Review by TheDoctorRAB — CHANGES_REQUESTED** [review]

**Comment by gonuke** [comment]
This looks like a great improvement - let me know if you're ready to merge (once the URL is updated)

File: CONTRIBUTING.md (hunk 1 above)
Comment (gonuke) [reply CONTRIBUTING.md]: Maybe this URL is better: https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows

**Comment by katyhuff** [comment]
Based on our conversation, I've now combined the node-criteria doc with the contribution doc. Based on these reviews  (thanks!) I've fixed the link. Ready to merge unless there are more review comments. Thanks!

**Comment by gonuke** [comment]
Thanks @katyhuff

**Pull request merged**